symengine/printers/latex.cpp

```diff
@@ -230,6 +239,61 @@ void LatexPrinter::bvisit(const StrictLessThan &x)
     str_ = s.str();
 }
 
+std::string LatexPrinter::apply2(const DenseMatrix &b)
+{
+    this->bvisit(b);
+    return str_;
+}
+
+void LatexPrinter::bvisit(const DenseMatrix &m) {
```

Comment:
You don't need to use `bvisit` or `apply`.

```suggestion
std::string latex(const DenseMatrix &x) {
```

Reply:
Ok, that simplies it a lot!